.circleci/config.yml

```diff
@@ -202,6 +209,19 @@ jobs:
           name: Deployment
           command: bundle exec fastlane android deploy
 
+  deploy-typescript:
+    executor: node/default
+    working_directory: ~/purchases-hybrid-common/typescript
+    steps:
+      - checkout:
+          path: ~/purchases-hybrid-common
```

Comment:
Is this needed?

Reply:
Not really... I think I just need to update the `working_directory` to be a relative path instead. I just copied this from somewhere, but that's probably cleaner 👍

Reply:
So after testing it out, I actually think it's needed... If I change the working_directory but not this, it causes errors when creating the folders. I could cd to the folder before the command... But feels that this is better. Lmk if you think otherwise!